roles/installer/ospd/overcloud/scale/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+- name: Set scaler variables
+  set_fact:
+    scale_service={{ item.key }}
+    scale_amount={{ item.value }}
+  when: (item.key == "ceph-storage") or
+        (item.key == "compute")
+  with_dict: "{{ hostvars[inventory_hostname] }}"
+
+- name: Creating scale-down script.
+  shell: "cp -pv  ~/overcloud_deploy.sh ~/overcloud_scale.sh"
+
+- name: change node amount
+  lineinfile:
```

Comment:
You probably want to use the "replace" module rather than "lineinfile"
http://docs.ansible.com/ansible/replace_module.html